src/pyskindose/settings_normalization.py

```diff
@@ -0,0 +1,159 @@
+
+class NormalizationSettings:
+    """A class to normalize RDSR for PySkinDose compliance.
+
+    Attributes
+    ----------
+    trans_offset : _TranslationOffset
+        See class variables of _TranslationOffset
+    trans_dir : _TranslationDirection
+        See class variables of _TranslationDirection
+    rot_dir : _RotationDirection
+        See class variables of _RotationDirection
+    field_size_mode : str
+        method for calculating field size at image receptor plane.
+        Choose either "CFA" (collimated field area) or "ACD" (actual shutter
+        distance). For more info, see calculate_field_size in geom_calc.py.
+    detector_side_length : str
+        side length of active image receptor area in cm.
+
+    """
+
+    def __init__(self, normalization_settings, data_parsed):
+        """Initialize class attributes."""
+        manufacturer = data_parsed['Manufacturer'][0]
```

Comment:
Using hard coded string for keys. Maybe extract to constants?

Reply:
done